src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +17,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
-    public boolean login(String email, String password) {
+
+    public boolean login(String email,String password) {
+        User myuser = userService.findByEmail(email);
+        if (myuser == null) {
+            return false;
+        }
+        if (password.equals(myuser.getPassword())) {
+            return true;
+        }
         return false;
```

Comment:
Let's simplify the code:
```suggestion
        return myuser != null && password.equals(myuser.getPassword());
```